src/components/reservation.css

```diff
@@ -41,7 +44,23 @@
 }
 
 @media (max-width: 600px) {
-  .reservation-list {
-    grid-template-columns: 1fr;
+  .reservation-page {
+    width: 100%;
+  }
+
+  .swiper-button-prev,
+  .swiper-button-next {
+    display: none;
+  }
+}
+
+@media (max-width: 768px) {
+  .reservation-page {
+    width: 100%;
+  }
+
+  .swiper-button-prev,
+  .swiper-button-next {
+    display: none;
```

Comment:
[OPTIONAL] 
The image gets very big in width when the screen size gets closer to tablet size. 
![image](https://github.com/Iqbal-Elham/Doctor-Reservation-frontend/assets/113791476/ea05e229-2344-4cbc-864c-061cf00467c0)

I would suggest you don't increase the image width, but instead give some margin to the container, so it is in the middle.

Reply:
your suggestion applies to
![photo fixt](https://github.com/Iqbal-Elham/Doctor-Reservation-frontend/assets/108926875/5f493178-1748-45b3-b4ce-64e12c1c82cd)